crates/adapter/src/fastly/core.rs

```diff
@@ -1689,6 +1689,27 @@ pub mod fastly_http_req {
             encodings.into(),
         ))
     }
+
+    #[export_name = "fastly_http_req#inspect"]
+    pub fn inspect(
+        ds_req: RequestHandle,
+        ds_body: BodyHandle,
+        info_mask: crate::bindings::fastly::api::types::InspectInfoMask,
+        info: *mut crate::bindings::fastly::api::types::InspectInfo,
+        buf: *mut u8,
+        buf_len: usize,
+        nwritten_out: *mut usize,
+    ) -> FastlyStatus {
+        alloc_result!(buf, buf_len, nwritten_out, {
+            fastly::api::http_req::inspect(
+                ds_req,
+                ds_body,
+                info_mask,
+                info,
```

Comment:
I think that `info` will need to be rebuilt as a `fastly::api::http_req::InspectConfig`. There's an example of the same over here: https://github.com/fastly/Viceroy/blob/a945b345b422df1c02db115368d44bafe7882f7a/crates/adapter/src/fastly/core.rs#L1460-L1488

Reply:
Thank you for pointing me to that! I've set up something similar for `InspectConfigOptions` and `InspectConfig` in af24988220b52134492a93d88e44727d3e9e6292.